host/4/publish.yml

```diff
@@ -638,6 +638,31 @@ jobs:
         docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.7-appservice
         docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.7-buildenv
 
+        docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8
```

Comment:
I think that you need to rebase on dev after this PR 
https://github.com/Azure/azure-functions-docker/pull/1156